pyproject.toml

```diff
@@ -3,7 +3,30 @@ requires = [
   'setuptools>=40.6.0',
   'wheel'
 ]
+build-backend = "setuptools.build_meta"
 
 [tool.black]
 line-length = 88
 python-version = ['py36']
+include = '\.pyi?$'
+exclude = '''
+
+(
+  /(
+      \.eggs         # exclude a few common directories in the
+    | \.git          # root of the project
+    | \.hg
+    | \.mypy_cache
+    | \.tox
+    | \.venv
+    | _build
+    | buck-out
+    | build
+    | dist
+    | documentation_builder
+    | release-notes
+    | scripts
```

Comment:
Actually it would be nice if the Python scripts here are maintained with black.

Reply:
Okay I'll remove `documentation_builder` and `scripts` from the list. `dist` is usually left out and `release-notes` doesn't have any Python files.

Reply:
Sorry, the UI was unspecific there. The comment was only on the line with `scripts`.

Reply:
No worries.